bin/startPipelineV2.sh

```diff
@@ -0,0 +1,409 @@
+#!/bin/bash
+
+#
+##
+### Environment and Bash sanity.
+##
+#
+if [[ "${BASH_VERSINFO}" -lt 4 || "${BASH_VERSINFO[0]}" -lt 4 ]]; then
+    echo "Sorry, you need at least bash 4.x to use ${0}." >&2
+    exit 1
+fi
+
+set -e # Exit if any subcommand or pipeline returns a non-zero exit status.
+set -u # Raise exception if variable is unbound. Combined with set -e will halt execution when an unbound variable is encountered.
+set -o pipefail # Fail when any command in series of piped commands failed as opposed to only when the last command failed.
+
+umask 0027
+
+# Env vars.
+export TMPDIR="${TMPDIR:-/tmp}" # Default to /tmp if $TMPDIR was not defined.
+SCRIPT_NAME="$(basename ${0} .bash)"
+INSTALLATION_DIR="$(cd -P "$(dirname "${0}")/.." && pwd)"
+LIB_DIR="${INSTALLATION_DIR}/lib"
+CFG_DIR="${INSTALLATION_DIR}/etc"
+HOSTNAME_SHORT="$(hostname -s)"
+ROLE_USER="$(whoami)"
+REAL_USER="$(logname)"
+
+#
+##
+### Functions.
+##
+#
+if [[ -f "${LIB_DIR}/sharedFunctions.bash" && -r "${LIB_DIR}/sharedFunctions.bash" ]]; then
+    source "${LIB_DIR}/sharedFunctions.bash"
+else
+    printf '%s\n' "FATAL: cannot find or cannot access sharedFunctions.bash"
+    trap - EXIT
+    exit 1
+fi
+
+function showHelp() {
+    #
+    # Display commandline help on STDOUT.
+    #
+    cat <<EOH
+===============================================================================================================
+Script to copy (sync) data from a succesfully finished analysis project from tmp to prm storage.
+
+Usage:
+
+    $(basename $0) OPTIONS
+
+Options:
+
+    -h   Show this help.
+    -g   Group.
+    -e   Enable email notification. (Disabled by default.)
+    -n   Dry-run: Do not perform actual sync, but only list changes instead.
+    -l   Log level.
+         Must be one of TRACE, DEBUG, INFO (default), WARN, ERROR or FATAL.
+
+Config and dependencies:
+
+    This script needs 3 config files, which must be located in ${CFG_DIR}:
+     1. <group>.cfg       for the group specified with -g
+     2. <host>.cfg        for this server. E.g.:"${HOSTNAME_SHORT}.cfg"
+     3. sharedConfig.cfg  for all groups and all servers.
+    In addition the library sharedFunctions.bash is required and this one must be located in ${LIB_DIR}.
+===============================================================================================================
+
+EOH
+    trap - EXIT
+    exit 0
+}
+
+
+#
+# Source config files.
+#
+
+generateScripts () {
+	_sampleType=$1 ## DNA or RNA
```

Comment:
* Local variables need to declare with local; without local in front of them they will be global instead.
* Please use ${var} (with curly brackets)
* Please quote variables.
E.g.:
     local _sampleType="${1}"